### PR TITLE
DOC: add API notes for jquery upgrade

### PR DESCRIPTION
**Pull request by tacaswell**
API notes for #5091

### CODE REVIEW

File: doc/api/api_changes.rst [hunk 1]
```diff
@@ -399,6 +399,18 @@ mpl.py
 Remove the module `matplotlib.mpl`.  Deprecated in 1.3 by
 PR #1670 and commit 78ce67d161625833cacff23cfe5d74920248c5b2
 
+bundled jquery
+--------------
+
+The version of jquery bundled with the webagg backend has been upgrade
```
Comment (jkseppan) [inline doc/api/api_changes.rst]: typo: should be "upgraded"

File: doc/api/api_changes.rst [hunk 2]
```diff
@@ -399,6 +399,18 @@ mpl.py
...
+from 1.7.1 to 1.11.3.  If you are using the version of jquery bundled
+with webagg you will need up update you html file as such
```
Comment (jkseppan) [inline doc/api/api_changes.rst]: typos: should be "need to update your html"

**Comment by jkseppan** [comment]
Aside from minor typos, I think this is correct.

**Comment by tacaswell** [comment]
@jkseppan Fixed and force pushed.

I also moved the section up above code removal as I am guessing more people will care about jquery.

**Comment by WeatherGod** [comment]
py2.7 test is failing due to some problem with Pandas

**Comment by jkseppan** [comment]
Either the tests or the Travis hosts have been pretty flaky lately. I restarted the failing job, since the failure is probably unrelated to this change.

**Comment by jkseppan** [comment]
The tests pass now. Let's merge this.

**Pull request merged**